docs/MockFunctionAPI.md

```diff
@@ -120,7 +120,7 @@ Clears all information stored in the [`mockFn.mock.calls`](#mockfnmockcalls), [`
 
 The [`clearMocks`](configuration#clearmocks-boolean) configuration option is available to clear mocks automatically before each tests.
 
-:::warning
+:::danger
```

Comment:
`:::warning` is a legacy Docusaurus admonition alias that is not documented, see https://docusaurus.io/docs/markdown-features/admonitions

Issue reported by Argos due to default label change: https://app.argos-ci.com/slorber/jest-website-visual-tests/builds/11/56882879

<img width="1290" alt="CleanShot 2023-09-01 at 16 48 20@2x" src="https://github.com/jestjs/jest/assets/749374/d498800c-2ff3-4708-9d74-182d77715c30">

<img width="1183" alt="CleanShot 2023-09-01 at 16 48 44@2x" src="https://github.com/jestjs/jest/assets/749374/303d9a85-a3fc-422d-ba02-1d40d53e07f9">

Using `:::danger` is the official red admonition name, keep the same label as before

---

Note: we are going to reintroduce `:::warning` soon, but it will be yellow instead of red (https://github.com/facebook/docusaurus/issues/7558)

Reply:
I think we wanted this to be yellow. That doesn't currently exist?

Reply:
The yellow admonition is currently `:::caution`.

I'd suggest to change that in another PR: the goal of this PR is to focus on having no regression and not improving things ;)

Reply:
This change can be moved to a preparation PR and would work with both MDX v1 and v2

Reply:
yellow is better!!

Reply:
landed that yesterday https://github.com/jestjs/jest/pull/14493